cves/kernel/CVE-2017-18255.yml

```diff
@@ -251,8 +249,8 @@ i18n:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: I say yes to this because this type of vulnerability will result in Denial of Service where the system will crash making it so the system will not be usable by people in general
```

Comment:
I think this may be false as the actual issue itself doesn't have anything to do with translated or international characters.